graphprot/Graph.py

```diff
@@ -17,22 +17,36 @@ def __init__(self):
         self.type = None
         self.name = None
         self.nx = None
-        self.score = {'irmsd': None, 'lrmsd': None,
+        self.score = {'irmsd': None, 'lrmsd': None, 'class': None,
                       'fnat': None, 'dockQ': None, 'binclass': None}
 
     def get_score(self, ref):
 
         ref_name = os.path.splitext(os.path.basename(ref))[0]
         sim = StructureSimilarity(self.pdb, ref)
-
+        
         self.score['lrmsd'] = sim.compute_lrmsd_fast(
-            method='svd', lzone=ref_name+'.lzone')
+            method='svd', lzone=ref_name+'.lzone') 
         self.score['irmsd'] = sim.compute_irmsd_fast(
-            method='svd', izone=ref_name+'.izone')
+            method='svd', izone=ref_name+'.izone') 
         self.score['fnat'] = sim.compute_fnat_fast()
         self.score['dockQ'] = sim.compute_DockQScore(
             self.score['fnat'], self.score['lrmsd'], self.score['irmsd'])
         self.score['binclass'] = self.score['irmsd'] < 4.0
+        
+        if self.score['irmsd'] < 1.0 :
+            _class = 1
+        elif self.score['irmsd'] < 2.0 :
+            _class = 2
+        elif self.score['irmsd'] < 4.0 :
+            _class = 3
+        elif self.score['irmsd'] < 6.0 :
+            _class = 4
+        else :
+            _class = 5
+        
+        self.score['class'] = _class
```

Comment:
do you need the `_class` variable ? why not 
```python
if self.score['irmsd'] < 1.0:
    self.score['class'] = 1
```

I also find `'class'` a bit generic. Maybe `'capri_class'` ?

Reply:
you could also shorten the code

```python
self.score['class'] = 5
for thr, val in zip([6.0, 4.0, 2.0, 1.0],[4,3,2,1]):
    if self.score['irmsd'] <= thr:
           self.score['class'] = val
```
but of course it's a bit less readable ... so up to you :) (plus I don't know if what I propose here works :D)